### ISSUE TITLE
Geocoding offered for sync table

### ISSUE THREAD

**Issue by juanignaciosl** [CartoDB/cartodb#5182]
I've created a URL sync table ([source](https://raw.githubusercontent.com/JJ/top-github-users-data/master/data/all-users-data.csv)) and geocoding is disabled at edit menu as usual, but when i switch to map view geocoding modal window appears and allow me to trigger a geocoding (although it won't work).

**Comment by xavijam** [comment]
I have a question, after fixing this issue I've just realized that we had in the old dialogs one modal saying, your dataset doesn't have any georef. data (as a warning dialog), but now we don't have that kind of modal because in the georeference dialog already says that with a yellow notification bar. Should we create a new one for this purpose, just warning the user? or not showing anything as It is happening now that I've fixed the bug?

@saleiva @viddo
